viewpump/src/main/java/io/github/inflationx/viewpump/ViewPump.java

```diff
@@ -23,13 +25,17 @@
     /** Use Reflection to intercept CustomView inflation with the correct Context. */
     private final boolean mCustomViewCreation;
 
+    /** A FallbackViewCreator used to instantiate a view via reflection when using the create() API. */
+    private static ReflectiveFallbackViewCreator mReflectiveFallbackViewCreator;
+
     private ViewPump(Builder builder) {
         interceptors = immutableList(builder.interceptors);
         List<Interceptor> interceptorsWithFallback = builder.interceptors;
         interceptorsWithFallback.add(new FallbackViewCreationInterceptor());
         mInterceptorsWithFallback = immutableList(interceptorsWithFallback);
         mReflection = builder.reflection;
         mCustomViewCreation = builder.customViewCreation;
+        mReflectiveFallbackViewCreator = new ReflectiveFallbackViewCreator();
```

Comment:
Thats true, a factory would be quite nice, similar to how the `ViewModelProvider` works, then we could in theory could provide dependencies at runtime (either via constructor/method etc).